ckanext/onicse_theme/i18n/ckanext-onicse_theme.pot

```diff
@@ -186,3 +174,18 @@ msgstr ""
 msgid "Go"
 msgstr ""
 
+#: ckanext/onicse_theme/templates/home/index.html:38
+msgid "Recently Updated Datasets"
+msgstr ""
```

Comment:
@teodora-aleksovska I guess we need to add translations for this and the empty strings below as well.